assets/src/edit-story/components/canvas/gridview/index.js

```diff
@@ -58,36 +58,67 @@ const RangeInputWrapper = styled.div`
   margin: 0 auto 75px auto;
 `;
 
-const Rectangle = styled.div`
+const Rectangle = styled.button.attrs(({ isDisabled }) => ({
+  disabled: isDisabled,
+}))`
+  border: 0;
+  padding: 0;
+  margin: 0;
+  min-width: unset;
+  background: transparent;
+  height: auto;
   display: flex;
   justify-content: center;
   align-items: center;
-  font-size: 32px;
+  cursor: pointer;
   color: ${({ theme }) => theme.colors.fg.v1};
   ${({ isLarge }) => (isLarge ? 'margin-left' : 'margin-right')}: 20px;
```

Comment:
The naming is a bit weird here. Why do large icons have left margin when small ones have right margin? Because of the actual design of course, but the code doesn't really explain this (I know it's in the original too).

Could we rewrite to a 20px `<Space />` being inserted between the components in stead?

Reply:
Not a huge fan of spacers, but makes sense to me here

Reply:
It's unfortunately the best way I've found to do this kind of spacing in a natural way. There's no _"margin-only-if-sibling-in-that-direction"_ property, though that would be super useful in many cases.

We could do `margin: 0 20px` and then accept that we'd have unnecessary margins that might impact responsive design, as the component is unnecessarily wide. Or we could circumvent that latter issue by adding negative margins to the wrapper, but then it gets even more complex.

Reply:
One bonus here is, that with spacers we get RTL for free (though I know styled has some internal way to flip left and right), but this gives us CSS-level RTL for free 😄.

Reply:
Great. Added SpacerSpan with space input props and default should be 8px. Maybe if we don't have this kind of component, we can move this to shared one I think.